src/main/java/teammates/ui/controller/InstructorFeedbackQuestionEditAction.java

```diff
@@ -80,46 +80,30 @@ private void editQuestion(FeedbackQuestionAttributes updatedQuestion) throws Inv
             statusToUser.add(new StatusMessage(err, StatusMessageColor.DANGER));
             isError = true;
         }
-        
-        if (updatedQuestion.questionNumber != 0) { // Question number was updated
-            List<String> questionDetailsErrors = updatedQuestion.getQuestionDetails().validateQuestionDetails();
-            List<StatusMessage> questionDetailsErrorsMessages = new ArrayList<StatusMessage>();
-            
-            for (String error : questionDetailsErrors) {
-                questionDetailsErrorsMessages.add(new StatusMessage(error, StatusMessageColor.DANGER));
-            }
-            
-            // if error is not empty not tested as extractFeedbackQuestionData method uses Assumptions to cover it
-            if (!questionDetailsErrors.isEmpty()) {
-                statusToUser.addAll(questionDetailsErrorsMessages);
-                isError = true;
-            } else {
-                logic.updateFeedbackQuestionNumber(updatedQuestion);
-                statusToUser.add(new StatusMessage(Const.StatusMessages.FEEDBACK_QUESTION_EDITED, StatusMessageColor.SUCCESS));
-            }
+
+        List<String> questionDetailsErrors = updatedQuestion.getQuestionDetails().validateQuestionDetails();
```

Comment:
This is not really your part of the code, but could you make a variable for <code>updatedQuestion.getQuestionDetails()</code>, since it seems to be used again below?